determined-ai/determined
# chore: include workloads in trial endpoint [DET-4036]

## Description

- use "workload" terminology instead of "step".
- checkout `proto/src/determined/trial/v1/trial.proto` for for the substitute list of steps.




## Test Plan

- call trials/id and check the payload and check the payload for workloads, best/latest validation, and best checkpoint
- call experiment/id/trials and check the payload for best/latest validation, and best checkpoint




## Commentary (optional)

slack discussion: https://determined-ai.slack.com/archives/C013B86TBPC/p1600892707004900




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234
